docs/source/1.0/spec/core/behavior-traits.rst

```diff
@@ -609,3 +609,30 @@ to request additional results from the operation.
    presence, absence, or contents of a continuation token.
 
 .. _UUID: https://tools.ietf.org/html/rfc4122
+
+
+.. _httpChecksumRequired-trait:
+
+--------------------------------------
+``httpChecksumRequired`` trait
+--------------------------------------
+
+Summary
+    Indicates that an operation requires the checksum present in its HTTP
```

Comment:
suggestion: "Indicates that an operation requires a checksum in its HTTP request. By default, the checksum used for a service is an MD5 passed in the `Content-MD5` header."

Reply:
Updated with feedback